scripts/biomedical/Human-GEM_Human1/scripts/format_genes.py

```diff
@@ -0,0 +1,57 @@
+"""
```

Comment:
Missing Google license and script header at the top

Reply:
added